curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -62,6 +62,7 @@ S3ClientAdaptorImpl::Init(
     chunkFlushThreads_ = option.chunkFlushThreads;
     maxReadRetryIntervalMs_ = option.maxReadRetryIntervalMs;
     readRetryIntervalMs_ = option.readRetryIntervalMs;
+    objectPrefix_ = option.objectPrefix;
```

Comment:
The acquisition of fsinfo is through https://github.com/opencurve/curve/blob/1f41ffe8c4f601d77f9aab302d5927ed4d8a6902/curvefs/src/client/curve_fuse_op.cpp#L88

The processing on the mds side is in
https://github.com/opencurve/curve/blob/1f41ffe8c4f601d77f9aab302d5927ed4d8a6902/curvefs/src/mds/mds_service.cpp#L199

mds needs to be assigned according to whether the optional prefixVersion exists.

Reply:
Hi, objectPrefix is inited in the [FuseopInit](https://github.com/opencurve/curve/blob/master/curvefs/src/client/fuse_s3_client.cpp#L63) process  ::curvefs::client::common::S3Info2FsS3Option(s3Info, &fsS3Option); And s3Info just come from FsInfo.

Reply:
> Hi, objectPrefix is inited in the [FuseopInit](https://github.com/opencurve/curve/blob/master/curvefs/src/client/fuse_s3_client.cpp#L63) process ::curvefs::client::common::S3Info2FsS3Option(s3Info, &fsS3Option); And s3Info just come from FsInfo.

FsInfo of FuseopInit come from 
https://github.com/opencurve/curve/blob/1f41ffe8c4f601d77f9aab302d5927ed4d8a6902/curvefs/src/client/curve_fuse_op.cpp#L88

Reply:
Hi, I think we shouldn't change the mds code, which require upgrading the binary. I can check whether the objectprefix has bee set in  the client, what do you think it is ok ?
`fsS3Opt->objectPrefix = s3.has_objectprefix() ? s3.objectprefix() : 0;`

Reply:
> Hi, I think we shouldn't change the mds code, which require upgrading the binary. I can check whether the objectprefix has bee set in the client, do you think it is ok ? `fsS3Opt->objectPrefix = s3.has_objectprefix() ? s3.objectprefix() : 0;`

LGTM